### DISCUSSION TITLE
HTML self closing tag not working

### DISCUSSION THREAD

**Discussion by ARR4NN** [remarkjs/remark#1131] [Q&A] [answered]
I am using an iframe like `<iframe width="auto" height="auto" src="https://www.youtube.com/embed/id" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen/>`

however anything after the iframe closes `/>` it will not render any more markdown
If i close the iframe with a <iframe> </iframe> it does work as intended

**Comment by remcohaszing** [comment, marked as answer]
HTML doesn't support self closing tags. There is a limited list of void elements though, which may be written as self closing tags. For more information, see https://developer.mozilla.org/en-US/docs/Glossary/Void_element.
